centaur/src/main/resources/reference.conf

```diff
@@ -57,7 +57,9 @@ centaur {
   # - centaur.test.Operations
   google {
     application-name = "centaur"
-    genomics.endpoint-url = "https://genomics.googleapis.com/"
+    genomics.endpoint-url = "https://lifesciences.googleapis.com/"
+    genomics.endpoint-url = ${?PAPI_ENDPOINT_URL}
```

Comment:
Where does `PAPI_ENDPOINT_URL` get set?  Is it documented somewhere?

Reply:
It gets set here: https://github.com/broadinstitute/cromwell/pull/5386/files#diff-3f478b5ac92d55b0df6c42f19e3eeb0cR120